CMakeLists.txt

```diff
@@ -164,7 +164,7 @@ IF(${ARCH64BIT} EQUAL 1)
 ELSE(${ARCH64BIT} EQUAL 1)
     SET(CPACK_RPM_PACKAGE_ARCHITECTURE "i386")
 ENDIF (${ARCH64BIT} EQUAL 1)
-SET(CPACK_SYSTEM_NAME "${CMAKE_SYSTEM_NAME}-${CPACK_RPM_PACKAGE_ARCHITECTURE}")
+SET(CPACK_SYSTEM_NAME "${CMAKE_SYSTEM_NAME}-x86_64")
```

Comment:
This will mislabel the ECL IDE installation package.
I think you want to make this change on line 174?

Reply:
Fixed